mzLib/Proteomics/ProteolyticDigestion/ProteolyticPeptide.cs

```diff
@@ -83,5 +84,22 @@ internal IEnumerable<PeptideWithSetModifications> GetModifiedPeptides(List<Modif
                 DictionaryPool.Return(twoBasedPossibleVariableAndLocalizeableModifications);
             }
         }
+
+        internal long numPossibleProteoforms(List<Modification> allKnownFixedModifications,
```

Comment:
This feels like it will be the entry point. This method should be public and well commented